src/training/baseline.py

```diff
@@ -1,10 +1,12 @@
 import torch
 from src.logger.simple import Logger
 from src.storage.simple import Storage
-from src.losses import ocr
+from src.losses.STRFL import OCRLoss
```

Comment:
Not sure we need to change it here -- in our paper we described that in first model we used old OCR loss.